Formula/stlink.rb

```diff
@@ -15,15 +15,13 @@ class Stlink < Formula
   end
 
   depends_on "libusb"
+  depends_on "gtk+3" => :optional
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  depends_on "cmake" => :build
```

Comment:
> Also, while I'm thinking about pet peeves, what in the world is the rule concerning

Is it worrying I instantly already have the answer to this in my mind? 😆.

The rule is generally if there's no formula `revision` there shouldn't be an empty line, if there is a formula `revision` there should be an empty line before the `head` to make it a little more obvious `revision`s don't apply to `head`.